server/views/pages/choose-practitioner.njk

```diff
@@ -8,96 +9,132 @@
 
 {% from "../components/pop/macro.njk" import pop %}
 
-{% set tableData = {
-    attributes: {
-    'data-module': 'moj-natural-sortable-table'
-    },
-    head: [
-        { text: 'Name',
-            attributes: {
-                "aria-sort": "none"
-            }
-        },
-        { text: 'Grade',
-            attributes: {
-                "aria-sort": "descending"
-            }
-        },
-        { text: 'Workload %',
-            format: "numeric",
-            attributes: {
-                "aria-sort": "none",
-                "aria-sort-natural": "ascending"
-            }
-        },
-        { text: 'Cases in past 7 days',
-            format: "numeric",
-            attributes: {
-                "aria-sort": "none"
-            }
-        },
-        { text: 'Community cases',
-            format: "numeric",
-            attributes: {
-                "aria-sort": "none"
-            }
-        },
-        { text: 'Custody cases',
-            format: "numeric",
-            attributes: {
-                "aria-sort": "none"
-            }
+{% set perTeamsTableData = [] %}
```

Comment:
These changes were made a while ago - I haven't needed to touch any of this since last week (except deleting the old code)